Bio/PDB/__init__.py

```diff
@@ -88,3 +97,67 @@
     from .SASA import ShrakeRupley
 except ImportError:
     pass
+
+_FormatToParser = {"pdb": PDBParser(), "cif": MMCIFParser()}
```

Comment:
This uses the same parser instance when multiple files are parsed. Are `PDBParser` and `MMCIFParser` reusable?

Reply:
As far as I can tell they are. Here in this example https://biopython.org/wiki/PDBParser the PDBParser is reused for benchmarking.

Reply:
I'd avoid this. We might want to pass arguments to the parsers via kwargs.